### ISSUE TITLE
Is /data really a data directory?

### ISSUE THREAD

**Issue by djmaze** [matrix-org/matrix-hookshot#896]
If I understand the documentation correctly, `/data` is just used for reading the configuration files and the passkey. Isn't it rather a _configuration directory_ then? It could also be mounted read-only if that is the case.

Would be good to know also for deploying this in a (swarm) cluster.

(By the way, the links to `helm/hookshot` in the documentation are broken.)

**Comment by MisguidedEmails** [comment]
Yes, `/data/config.yml`, `/data/passkey.pem` `/data/registration.yml` can all be read only. Hookshot itself does not need any persistent mounts.

**Comment by Half-Shot** [comment]
Yep, we should document this as such. I believe `/data` was something we just used early on and never changed. We could rename it to `/config` to keep it simple.
